app/views/resources/index.html.erb

```diff
@@ -14,6 +14,21 @@
         <% end %>
       </ul>
     </div>
+
+    <div class="dropdown">
+      <button class="btn btn-secondary dropdown-toggle" type="button" id="allResourcesButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
I did what you suggested but for some reason I need to place the .yellow-button class within .dropdown otherwise the color of the button's text will appear black when hovered over. Let me know if there is any problem.
```   
   .dropdown {
      display:block;
      margin: 0 auto; 

      .yellow-button {
        color: white;
        background-color: $yellow;
        font-size: 18px;
        font-family: ArcherBook;
        margin-bottom: 10px;
        margin-left: 5px;
      }
```